website/src/components/IndexPage/getOrganismStatistics.ts

```diff
@@ -2,8 +2,9 @@ import { DateTime, FixedOffsetZone } from 'luxon';
 
 import { LapisClient } from '../../services/lapisClient.ts';
 import { RELEASED_AT_FIELD, VERSION_STATUS_FIELD, IS_REVOCATION_FIELD } from '../../settings.ts';
-import { siloVersionStatuses } from '../../types/lapis';
+import { versionStatuses } from '../../types/lapis';
 
+// Excluding sequences whose latest version is revoked
```

Comment:
If we want this clarification it would be clearer next to `totalSequences` (for recent sequences its actually a bit more complicated). I would maybe just leave this to the code.

Reply:
And actually calling a latest version "revoked" is maybe confusing - I think you mean a "latest version is a revocation"

Reply:
happy with this - just curious what you mean by "for recent sequences it's more complicated"

Reply:
For recent sequences we only count version 1 to only count truly "new" sequences. Then we exclude version 1s which have since been revoked. If any of those have since been reinstated, then their "latest version" is not a revocation. But we still exclude them.

Reply:
Yeah going through the code carefully I realized this - we should document that we are aware of this issue in the code and state why we chose this (to avoid complicated 2-step query with accession lists)

Reply:
Thanks for making me realize the complexity, adding a comment later to explain the tradeoff and potential for edge cases